src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -135,6 +135,8 @@ namespace InternalHeatGains {
     int constexpr ITEClassA4(4);
     int constexpr ITEClassB(5);
     int constexpr ITEClassC(6);
+    int constexpr ITEClassH1(7);
```

Comment:
From the department of "you touched it, you fix it (even if you didn't break it)" :grin: Take this opportunity to convert these to an `enum class` and the code below to `getEnumerationValue`.

Reply:
@amirroth Thanks! These few comments has been addressed in the new commits that were just pushed in.